extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -14,21 +14,16 @@ import { convertToGibibyteString } from '../../../common/utils';
 
 export type ConfigurationSpecModel = {
 	workers?: number,
-	workerCoresRequest?: string,
-	workerCoresLimit?: string,
-	workerMemoryRequest?: string,
-	workerMemoryLimit?: string,
-	coordinatorCoresRequest?: string,
-	coordinatorCoresLimit?: string,
-	coordinatorMemoryRequest?: string,
-	coordinatorMemoryLimit?: string
+	coresRequest: { w?: string, c?: string },
```

Comment:
Would suggest : 

1. Creating a separate type for `{ w?: string, c?: string}` so it's easier to update/reference
2. Using the full names `worker` and `coordinator` so it's clear what they are